packages/matrix-identity-server/src/db/sql/pg.ts

```
@@ -60,18 +60,58 @@ class Pg<T extends string> extends SQL<T> implements IdDbBackend<T> {
               opts.host = RegExp.$1
               opts.port = parseInt(RegExp.$2)
             }
+            const sslType =
+              opts.ssl == null || opts.ssl === false
```

Biome (2.4.9)

[error] Using == may be unsafe if you are relying on type coercion.

(lint/suspicious/noDoubleEquals)